include/rmm/mr/device/cuda_async_view_memory_resource.hpp

```diff
@@ -17,9 +17,9 @@
 
 #include <rmm/cuda_device.hpp>
 #include <rmm/cuda_stream_view.hpp>
-#include <rmm/detail/dynamic_load_runtime.hpp>
 #include <rmm/detail/error.hpp>
 #include <rmm/detail/export.hpp>
+#include <rmm/detail/runtime_async_alloc.hpp>
```

Comment:
Is this include being used in this file?

Reply:
Not anymore, removed.